docs/pages/en-us/Architecture.md

```diff
@@ -5,78 +5,216 @@ title: Architecture
 banner:
     icon: 'assets/img/auditing-graphic.png'
     title: Auditing
-    subtitle: Traceability, accountability, analytics, archiving and reporting of application tenant events.
+    subtitle: Traceability, accountability and archiving of application tenant events.
     links:
         - title: GitHub
           url: https://github.com/CAFAudit/audit-service
 ---
 
 # Architecture
 
-Auditing logs audit events on a per application, per tenant basis.  The Audit Management web service facilitates the addition of new applications and new tenants.
+The Audit library logs audit events on a per tenant basis and facilitates the addition of new tenants and new applications.
```

Comment:
Reworded to "The Audit library logs audit event metadata, belonging to tenant applications, into Elasticsearch." in the latest commit.